---
permalink: /
title: ""
excerpt: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% if site.google_scholar_stats_use_cdn %}
{% assign gsDataBaseUrl = "https://cdn.jsdelivr.net/gh/" | append: site.repository | append: "@" %}
{% else %}
{% assign gsDataBaseUrl = "https://raw.githubusercontent.com/" | append: site.repository | append: "/" %}
{% endif %}
{% assign url = gsDataBaseUrl | append: "google-scholar-stats/gs_data_shieldsio.json" %}

<span class='anchor' id='about-me'></span>

I work at L3 Bioinformatics Limited as a senior research scientist in Hong Kong now. I am now working on in area of bioinformatics algorithm, deep learning, and genomics.

I received my Ph.D. degree from the [Department of Computer Science, the University of Hong Kong](https://www.cs.hku.hk/) in September 2023, supervised by Prof. [Ruibang Luo](http://www.bio8.cs.hku.hk/). Prior to that, I received my M. Eng. and B. Eng. degree in Software Engineering at [School of Computer Science and Engineering, Sun Yat-sen University](https://cse.sysu.edu.cn/).

My research interest includes variant calling, somatic variant detection, third generation sequencing data analysis, etc. I have published 10+ papers in peer-reviewed journals and conferences, including Nature Computational Science, Briefings in Bioinformatics, and BMC Bioinformatics.


# 🔥 News
- 2023.10 &nbsp;🎉🎉 I joined L3 Bioinformatics Limited as a senior research scientist. 
- 2023.09 &nbsp;🎉🎉 I graduated from the Department of Computer Science, the University of Hong Kong. 
- 2023.08: &nbsp;🎉🎉 A preprint describing ClairS's algorithms and results is at [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.08.17.553778v1).
- 2023.05: &nbsp;🎉🎉 ClairS was supported as official somatic small variant caller tool by Oxford Nanopore Technologies, link [here](https://labs.epi2me.io/colo-2023.05/).
- *2023.04*: &nbsp;🎉🎉 I gave a talk in Recomb-Seq on our new somatic variant caller [ClairS](https://github.com/HKU-BAL/ClairS). 
- *2023.02*: &nbsp;🎉🎉 Our paper [Clair3](https://www.nature.com/articles/s43588-022-00387-x) was accepted by Nature Computational Science.
- *2021.10*: &nbsp;🎉🎉 Our first bioinformatics software [Clair3](https://github.com/HKU-BAL/Clair3) was supported as official variant caller tool by Oxford Nanopore Technologies, link [here](https://labs.epi2me.io/gm24385_q20_2021.10/).

# 📝 Publications 

Please check my google scholar [here](https://scholar.google.com/citations?user=NBH39WAAAAAJ&hl=zh-CN&oi=sra).

# 🎖 Honors and Awards
- *2019-2023* University Postgraduate Fellowships (4 students in 2019)
- *2019-2023* Postgraduate Scholarships
- 2018 First-class Graduate Academic Scholarship (Top 1%)
- 2014-2016 First-class Undergraduate Academic Scholarship (Top 1%)

# 📖 Educations
- 2019.09 - *2023.09*, Ph.D., Department of Computer Science, The University of Hong Kong, Hong Kong.
- *2017.09 - 2019.06*, M. Eng., School of Computer Science and Engineering, Sun Yat-sen University, Guangzhou.
- *2013.09 - 2017.06*, B. Eng., School of Computer Science and Engineering, Sun Yat-sen University, Guangzhou.

# 💻 Internships
- *2018.05 - 2018.09*, Summer intern, Intelligent Recommendation Center, Tencent.
- *2017.06 - 2017.09*, Research intern, Medical image group, Sun Yat-sen University Cancer Center.
